CHANGELOG.md

```diff
@@ -10,46 +10,55 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- **Breaking change**: Changed the actions of the <kbd>Shift</kbd> + <kbd>Page Up</kbd>/<kbd>Page Down</kbd> keyboard shortcuts, to match the usual spreadsheet software behavior. [#9365](https://github.com/handsontable/handsontable/issues/9365)
+- Changed two scripts of the main Handsontable workspace (`./`), to speed up the build process: now, the `npm run build` and `npm run test` scripts don't build or test the Handsontable examples (`./examples`). [#9412](https://github.com/handsontable/handsontable/issues/9412)
+- Changed the version of the Moment.js dependency from to 2.24.0 to 2.29.3, in reaction to a minor security vulnerability found recently in Moment.js. [#9381](https://github.com/handsontable/handsontable/issues/9381)
 
-### Added
-- **Breaking change**: Introducing the ShortcutManager - a new API for handling shortcuts. [#8942](https://github.com/handsontable/handsontable/issues/8942)
-- Added a new configuration option: `fixedColumnsStart` [#8760](https://github.com/handsontable/handsontable/issues/8760)
-- Added Cmd/Ctrl + Enter keyboard shortcut that allows to fill the selected range of cells with the value of the active cell. [#9366](https://github.com/handsontable/handsontable/issues/9366)
-- Adds support for Home/End and fix buggy behavior of the Cmd/Ctrl+Z/Shift+Z keyboard shortcuts. [#9367](https://github.com/handsontable/handsontable/issues/9367)
-- Add Arabic Language [#9208](https://github.com/handsontable/handsontable/issues/9208)
+### Fixed
+- Fixed an issue where cell filtering used formula results instead of source values. [#5455](https://github.com/handsontable/handsontable/issues/5455)
+- Fixed an issue where the Handsontable documentation didn't build locally on Windows. [#8677](https://github.com/handsontable/handsontable/issues/8677)
+- Fixed a wrong TypeScript definition in the `BasePlugin` class. [#9175](https://github.com/handsontable/handsontable/issues/9175)
+- Fixed an issue where using `updateSettings()` caused configuration options to disappear. [#6552](https://github.com/handsontable/handsontable/issues/6552)
+- Fixed an issue where using `updateSettings()` caused hidden columns to reappear. [#7165](https://github.com/handsontable/handsontable/issues/7165)
+- Fixed an issue where using `updateSettings()` caused merged cells to unmerge. [#3315](https://github.com/handsontable/handsontable/issues/3315)
+- Fixed an issue where using `updateSettings()` caused the state of nested rows to reset. [#8838](https://github.com/handsontable/handsontable/issues/8838)
+- Fixed an issue where using `updateSettings()` caused problems with column sorting. [#7688](https://github.com/handsontable/handsontable/issues/7688)
+- Fixed an issue where the <kbd>Ctrl</kbd>/<kbd>Cmd</kbd> + <kbd>Z</kbd> and <kbd>Ctrl</kbd>/<kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>Z</kbd> keyboard shortcuts didn't behave properly inside cell editors. [#9367](https://github.com/handsontable/handsontable/issues/9367)
```

Comment:
I am not sure if it should be listed here, it depends on the answer in https://github.com/handsontable/handsontable/issues/9367#issuecomment-1109778441

Reply:
Leaving as it is, as per [this comment](https://github.com/handsontable/handsontable/issues/9367#issuecomment-1109787067)